scripts/functions/generate_data.py

```diff
@@ -593,7 +606,7 @@ def add_experiments(
 
             for j in range(i):
                 # Scalar to shift gene expressiond data
-                stretch_factor = np.random.normal(0.0, 0.1, [1, num_genes])
+                stretch_factor = np.random.normal(0.0, 0.2, [1, num_genes])
```

Comment:
any reason for shifting the stretch factor?

Reply:
This was changed when I was comparing the results of the svcca curve for analysis_0 and analysis_1 -- I hypothesized that the results of analysis_1 might be because the magnitude of the added technical noise was lower.  In order to test this I lowered the magnitude of the noise for analysis_0 to see if I got a similar result as analysis_1.  I did not.  I think there is something else going on that I need to think more about.

Reply:
Hmm... yes, worth thinking more about indeed.